test/cloudwatchlogs/publish_logs_test.go

```diff
@@ -332,3 +497,154 @@ func checkData(t *testing.T, start time.Time, lineCount int) {
 	)
 	assert.NoError(t, err)
 }
+
+func ValidateEntity(t *testing.T, logGroup, logStream string, begin, end *time.Time, expectedEntity expectedEntity) {
+	log.Printf("Validating entity for log group: %s, stream: %s", logGroup, logStream)
+
+	logGroupInfo, err := getLogGroup(logGroup)
+	for _, lg := range logGroupInfo {
+		if *lg.LogGroupName == logGroup {
+			log.Println("Log group " + *lg.LogGroupName + " exists")
+			break
+		}
+	}
+	assert.NoError(t, err)
+	
+	log.Printf("Query start time is " + begin.String() + " and end time is " + end.String())
+	queryId, err := getLogQueryId(logGroup, begin, end)
+	assert.NoError(t, err)
+	log.Printf("queryId is " + *queryId)
+	result, err := getQueryResult(queryId)
```

Comment:
`getLogQueryId` and `getQueryResult` are not needed anymore since we can just use default client now. There's a helper function here: https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/util/awsservice/cloudwatchlogs.go#L202